prime-router/docs/getting_started.md

```diff
@@ -42,38 +42,12 @@ You will need to have at least the following pieces of software installed _local
 * [OpenJDK](getting-started/install-openjdk.md) (currently targetting 11 through 15)
 * [Azure Functions Core Tools](getting-started/install-afct.md) (currently targetting 3)
 
-Optionally, you may also want to install:
+Optionally, you may or will probably also want to install at least some of:
```

Comment:
Too passive voice and confusing.  Just say something like: The following tools may be installed for development purposes